docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/authorize-prisma-cloud.adoc

```diff
@@ -223,19 +223,19 @@ The authentication response of the app will be returned to this URI.
 +
 The client secret is a secret string that the application uses to prove its identity when requesting a token.
 +
-.. Select menu:Certificates{sp}&{sp}secrets[+ New client secret].
+.. Select *Certificates{sp}&{sp}secrets[+ New client secret*.
```

Comment:
The correct syntax is *Certificates & secrets > + New client secret*.
The {sp} must be removed from the menu: format. Please fix all the instances.